### PR TITLE
Capture horizon config file

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

File: suse_openstack_cloud [hunk 1]
```diff
@@ -134,6 +136,8 @@ _capture_filter() {
         # "bmc_password": "s3kr1t",
         # "machine-install": { "password": "s3kr1t" },
         -e 's/\("\([a-z_]\+\)\?\(passw\(or\)\?d\|token\)":\? \+"\+\)[^"]\+"/\1'"$CENSORED"'"/gi'
+        # 'password': 's3kr1t',
+        -e "s/\('\([a-z_]\+\)\?\(passw\(or\)\?d\|token\)':\? \+'\+\)[^']\+'/\1$CENSORED'/gi"
```
Comment (aplanas) [inline suse_openstack_cloud]: I guess that is expected to fail in case of escaped strings, like `'x\'x'`. The other regex have the same problem, IIUC.

**Comment by aplanas** [comment]
+1

File: suse_openstack_cloud (hunk 1 above)
Comment (vuntz) [inline suse_openstack_cloud]: I guess so. I really just took the other regexp and changed the quotes (as changing the other regexp to deal with both style of quotes was making it really unreadable)

File: suse_openstack_cloud [hunk 2]
```diff
@@ -93,6 +93,7 @@ _capture() {
 _filter_test_cases() {
     cat <<EOF
     "password": "s3kr1t",
+    'password': 's3kr1t',
```
Comment (nicolasbock) [inline suse_openstack_cloud]: Why do we need to change the quote style here?

Comment (toabctl) [inline suse_openstack_cloud]: It's not changing it - it's adding another testcase afaics

File: suse_openstack_cloud [hunk 3]
```diff
@@ -320,6 +324,8 @@ for svc in keystone rabbitmq glance swift nova cinder neutron heat ceilometer ma
     esac
 done
 
+pconf_files /srv/www/openstack-dashboard/openstack_dashboard/local/local_settings.py
```
Comment (toabctl) [inline suse_openstack_cloud]: I think we also need all files from ```/srv/www/openstack-dashboard/openstack_dashboard/local/local_settings.d/```

Comment (vuntz) [inline suse_openstack_cloud]: Good catch, done.

**Comment by toabctl** [comment]
+1

**Comment by aplanas** [comment]
still +1

File: suse_openstack_cloud (hunk 2 above)
Comment (nicolasbock) [inline suse_openstack_cloud]: Oops, yes, you are right.

**Pull request merged**